executor/executor.cc

```diff
@@ -113,11 +113,35 @@ static void reply_handshake();
 #endif
 
 #if SYZ_EXECUTOR_USES_SHMEM
-const int kMaxOutput = 16 << 20;
+// The output region is the only thing in executor process for which consistency matters.
+// If it is corrupted ipc package will fail to parse its contents and panic.
+// But fuzzer constantly invents new ways of how to currupt the region,
+// so we map the region at a (hopefully) hard to guess address with random offset,
+// surrounded by unmapped pages.
+// The address chosen must also work on 32-bit kernels with 1GB user address space.
+const uint64 kOutputBase = 0x1b2bc20000ull;
+
+#if SYZ_EXECUTOR_USES_FORK_SERVER
+// Allocating (and forking) virtual memory for each executed process is expensive, so we only mmap
+// the amount we might possibly need for the specific received prog.
+const int kMaxOutputComparisons = 14 << 20; // < 1% of all executions.
```

Comment:
"1% of all executions" what?
should this be "enough for 99% of all executions"?

Reply:
If formulated that way, it should be "enough for 100-epsilon% executions". The idea was to underline that we mmap such a big output buffer in only ~1% of all cases. `kMaxOutputCoverage` will be mmapped in ~30% of all cases.

Reply:
Reformulated those comments to make them more clear